runner.sh

```diff
@@ -59,9 +59,16 @@ export_drawio_pages() {
   local filename="$3"
 
   local pagenum=0
+
+  local numberOfPage=$(sgrep -c '"name=\"".."\""' "$path")
```

Comment:
prefer the snake case notation for variable name

Reply:
done